docs/create-osiam-connector.md

```diff
@@ -24,3 +60,20 @@ define them for **all** connectors you create. This will be addressed in a
 future version, but note that it is recommended to use only **one** connector
 instance for the whole application, unless you need to use different OAuth
 clients.
+
+## Legacy Schemas
+
+(since 1.8) You can configure the use of legacy schemas, i.e. schemas that were defined before SCIM
```

Comment:
I don't like starting sentences with parentheses. What about "Starting with the connector version 1.8 you can configure the use of legacy schemas".